### PR TITLE
Major appliance sat

### PR DESCRIPTION
**Pull request by afontani**
Companion PR: <https://github.com/NREL/resstock-estimation/pull/15>

## Pull Request Description

The purpose of this pull request is to reduce the number of appliances in multifamily units. The current assumption is that all units have Dishwashers and Clothes Washers. However multifamily units should have a lower probability of having laundry facilities (in unit) and extra refrigerators and stand-alone freezers. As a result, building type dependency was added to these characteristics and the fraction of units without the appliances is adjusted.

### Adding a building type dependency
A dependency to Geometry Building Type RECS is added to the following housing characteristics:

- Clothes Washer Presence
- Dishwasher
- Clothes Dryer
- Refrigerator
- Misc Extra Refrigerator
- Misc Extra Freezer

### Updating Refrigeration Efficiency Levels
A combination of RECS 2009 and shipment-weighted efficiencies by year is used to map the age of the refrigerators to efficiency levels. The following housing characteristics were updated with these two data sources.

- Refrigerator
- Misc Extra Refrigerator
- Misc Extra Freezer (RECS 2009 only)

### New use of the American Housing Survey (AHS)
American Housing Survey (AHS) data was used to create saturations for the following housing characteristics. The benefit of using AHS is the saturation can be specified by Census Division.

- Clothes Washer Presence
- Clothes Dryer

### AHS Region spatial field
This new field uses geographic regions specified by AHS. The field uses the Top 15 CBSAs in U.S.

1. Atlanta-Sandy Springs-Roswell, GA
2. Boston-Cambridge-Newton, MA-NH'
3. Chicago-Naperville-Elgin, IL-IN-WI
4. Dallas-Fort Worth-Arlington, TX
5. Detroit-Warren-Dearborn, MI
6. Houston-The Woodlands-Sugar Land, TX
7. Los Angeles-Long Beach-Anaheim, CA
8. Miami-Fort Lauderdale-West Palm Beach, FL
9. New York-Newark-Jersey City, NY-NJ-PA
10. Philadelphia-Camden-Wilmington, PA-NJ-DE-MD
11. Phoenix-Mesa-Scottsdale, AZ
12. Riverside-San Bernardino-Ontario, CA
13. San Francisco-Oakland-Hayward, CA
14. Seattle-Tacoma-Bellevue, WA
15. Washington-Arlington-Alexandria, DC-VA-MD-WV

Each of these CBSA are defined by county according to the February 2013 delineation file (https://www.census.gov/geographies/reference-files/time-series/demo/metro-micro/delineation-files.html). For counties not in these defined CBSAs, they are assigned their associated Census Division.

### TSV Specific Sources Tagging
The current workflow of the tsv_maker.py created for different data sources had the same Created by line in each tsv the maker writes. Since different TSVs may use different data sources for each file, the methods were updated to write a set of sources for a specific file in the following form.
```
# Created by: ...
# Source: ...
# Source: ...
...
```

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [x] Any new options are added to `project_testing`
- [x] `project_testing` runs without any failures
- [x] No unexpected regression test changes
- [x] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Comment by ejhw** [comment]
>A combination of RECS 2009 and Home Energy Saver (HES) sales shares

Isn't it shipment-weighted efficiency by year and not sales share from HES?
How about:
> A combination of RECS 2009 refrigerator age and Home Energy Saver (HES) shipment-weighted efficiency by year  is used to develop a distribution of refrigerators efficiency levels.

**Comment by afontani** [comment]
@ejhw You are correct. I will make the changes.

File: project_multifamily_beta/housing_characteristics/Dishwasher.tsv [hunk 1]
```diff
@@ -1,5 +1,202 @@
-Dependency=Usage Level	Option=None	Option=318 Rated kWh, 80% Usage	Option=318 Rated kWh, 100% Usage	Option=318 Rated kWh, 120% Usage	Option=290 Rated kWh, 80% Usage	Option=290 Rated kWh, 100% Usage	Option=290 Rated kWh, 120% Usage
-Low	0.000000	0.360000	0.000000	0.000000	0.640000	0.000000	0.000000
-Medium	0.000000	0.000000	0.360000	0.000000	0.000000	0.640000	0.000000
-High	0.000000	0.000000	0.000000	0.360000	0.000000	0.000000	0.640000
-Average	0.000000	0.000000	0.360000	0.000000	0.000000	0.640000	0.000000
+Dependency=Geometry Building Type RECS	Dependency=Location Region	Dependency=Usage Level	Option=290 Rated kWh, 100% Usage	Option=290 Rated kWh, 120% Usage	Option=290 Rated kWh, 80% Usage	Option=318 Rated kWh, 100% Usage	Option=318 Rated kWh, 120% Usage	Option=318 Rated kWh, 80% Usage	Option=None	sample_count	sample_weight
+Mobile Home	CR02	Average	0.060507	0.000000	0.000000	0.213799	0.000000	0.000000	0.725693	7.250000	63468.816406
```
Comment (ejhw) [inline project_multifamily_beta/housing_characteristics/Dishwasher.tsv]: Side note: Not necessary for this PR, but we can remove `Average`. It hasn't been used in a long time and is functionally the same as `Medium`.

File: resources/options_lookup.tsv [hunk 2]
```diff
@@ -1668,7 +1668,7 @@ Cooking Range	"Electric, 100% Usage"	ResidentialApplianceCookingRange	fuel_type=
...
-Cooking Range Schedule	National Average	ResidentialApplianceCookingRange	"weekday_sch=0.007, 0.007, 0.004, 0.004, 0.007, 0.011, 0.025, 0.042, 0.046, 0.048, 0.042, 0.050, 0.057, 0.046, 0.057, 0.044, 0.092, 0.150, 0.117, 0.060, 0.035, 0.025, 0.016, 0.011"	"weekend_sch=0.007, 0.007, 0.004, 0.004, 0.007, 0.011, 0.025, 0.042, 0.046, 0.048, 0.042, 0.050, 0.057, 0.046, 0.057, 0.044, 0.092, 0.150, 0.117, 0.060, 0.035, 0.025, 0.016, 0.011"	"monthly_sch=1.097, 1.097, 0.991, 0.987, 0.991, 0.890, 0.896, 0.896, 0.890, 1.085, 1.085, 1.097"																																										
+Cooking Range Schedule	National Average	ResidentialApplianceCookingRange	"weekday_sch=0.01490, 0.0120, 0.0102, 0.0096, 0.0107, 0.0187, 0.0306, 0.0355, 0.0356, 0.0341, 0.0378, 0.0438, 0.0451, 0.0413, 0.0413, 0.0586, 0.0942, 0.1252, 0.1162, 0.0802, 0.0448, 0.0239, 0.0191, 0.0166"	"weekend_sch=0.01320, 0.0109, 0.0091, 0.0083, 0.0085, 0.0109, 0.0213, 0.0368, 0.0493, 0.0497, 0.0522, 0.0564, 0.0568, 0.0520, 0.0513, 0.0631, 0.0862, 0.1070, 0.0979, 0.0685, 0.0395, 0.0212, 0.0167, 0.0132"	"monthly_sch=1.097, 1.097, 0.991, 0.987, 0.991, 0.890, 0.896, 0.896, 0.890, 1.085, 1.085, 1.097"
```
Comment (ejhw) [inline resources/options_lookup.tsv]: Unintentional commit?

Comment (afontani) [reply resources/options_lookup.tsv]: This was in response to the Diversify cooking schedules analysis JIRA LP-548 (https://agile.nrel.gov/browse/LP-548). We decided to use schedules informed by the submeter data and the transferability study. At least until the stochastic occupant model is pulled into master.

File: project_multifamily_beta/housing_characteristics/Dishwasher.tsv (hunk 1 above)
Comment (afontani) [reply project_multifamily_beta/housing_characteristics/Dishwasher.tsv]: Thank you, I always wanted to do this.

**Comment by afontani** [comment]
@joseph-robertson @ejhw @JLReyna Let me know if there are any comments about this PR.

**Pull request merged**